chat-widget/src/common/facades/FacadeChatSDK.ts

```diff
@@ -131,6 +134,11 @@ export class FacadeChatSDK {
 
     private async tokenRing(): Promise<PingResponse> {
 
+        if (this.isFacaDisabled === true){
```

Comment:
Would it make more sense to use `disableReauthentication` instead then we internally disable the facade?. In my opinion, contributors may know this props `facade` means internally but not for people outside of it.

Reply:
done